sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/SystemEventMappings.java

```diff
@@ -3,6 +3,16 @@
 
 package com.azure.messaging.eventgrid;
 
+import com.azure.messaging.eventgrid.systemevents.ACSChatMemberAddedToThreadWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMemberRemovedFromThreadWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageDeletedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageEditedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatMessageReceivedEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadCreatedWithUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadPropertiesUpdatedPerUserEventData;
+import com.azure.messaging.eventgrid.systemevents.ACSChatThreadWithUserDeletedEventData;
+import com.azure.messaging.eventgrid.systemevents.AcssmsDeliveryReportReceivedEventData;
+import com.azure.messaging.eventgrid.systemevents.AcssmsReceivedEventData;
```

Comment:
It's odd that some classes use Pascal case and some use all uppercase letters for ACS. Can we use Pascal case throughout?